stable/mysql/README.md

```diff
@@ -35,7 +35,7 @@ You can retrieve your root password by running the following command. Make sure
 To uninstall/delete the `my-release` deployment:
 
 ```bash
-$ helm delete my-release
+$ helm delete --purge my-release
```

Comment:
I disagree with this change. `--delete` is generally not specified in the readme.